manim/renderer/opengl_renderer.py

```diff
@@ -353,25 +357,28 @@ def init_scene(self, scene):
             scene.__class__.__name__,
         )
 
+    def update_skipping_status(self):
+        """
+        This method is used internally to check if the current
+        animation needs to be skipped or not. It also checks if
+        the number of animations that were played correspond to
+        the number of animations that need to be played, and
+        raises an EndSceneEarlyException if they don't correspond.
+        """
+        if config["from_animation_number"]:
+            if self.num_plays < config["from_animation_number"]:
+                self.skip_animations = True
+        if config["upto_animation_number"]:
+            if self.num_plays > config["upto_animation_number"]:
+                self.skip_animations = True
+                raise EndSceneEarlyException()
+
+    @pass_scene_reference
```

Comment:
I think we found that pass_scene_reference isn't needed here.

Reply:
I deleted it from OpenGL renderer and everything still works as expected. Please note that the decorator still exists in the Cairo renderer.